### PR TITLE
Add linter

### PR DESCRIPTION
**Pull request by shahedakhatoon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
+   - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+3. When you open your first pull request you should see the result of the GitHub actions checks:
+
+![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+
```
Comment (elmar8287) [inline README.md]: - [ ] Please, consider using the [MICROVERSE template](https://github.com/microverseinc/readme-template) for README file. You used the information regarding linter. The README file should give information and description about your project: project title and description, information about the author, how to set up and etc. All this information you can find in link I marked to the MICROVERSE template for the README file.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```
Comment (elmar8287) [inline style.css:3]: - [ ] Pease, consider giving a class name for the h1 tag and then style it, instead directly styling the HTML tag.

**Review by elmar8287 — COMMENTED** [review]
Hi @shahedakhatoon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the comments under the review._
- [ ] Please, in the future, do not merge the branch before getting approved for PR. It will make you more professional your submission.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (shahedakhatoon) [reply README.md]: @elmar8287  Please see the changes I've made in the README file.

**Pull request merged**